assets/js/dashboard.js

```diff
@@ -0,0 +1,32 @@
+$ = jQuery;
+
+// Dashboard page.
+$(function() {
+	$('.btn-cp-action').off('click').on('click', function (event) {
+		event.preventDefault();
+		var $btn      = $(this);
+		var $response = $( '.cp-subitem-response', $btn.parent() );
+
+		$.ajax({
+			url: ajaxurl,
```

Comment:
`ajaxurl` is undefined here. Is it defined outside this event, should it be built in this function from the DOM, or passed as an event parameter? https://stackoverflow.com/questions/3273350/jquerys-click-pass-parameters-to-user-function#9467172

Reply:
Since WP 2.8 ajaxurl is a global js variable defined by WP itself and points to admin-ajax.php (this is true only for use in the WP admin side) https://codex.wordpress.org/AJAX_in_Plugins

Reply:
Oh I see, that's interesting, thanks.